app/src/main/res/values-ar/strings.xml

```diff
@@ -106,6 +106,7 @@
     <string name="app_components_common_label_label">التسمية</string>
     <string name="app_components_common_onColoredBackground_label">على خلفية ملونة</string>
     <string name="app_components_common_icon_a11y">أيقون</string>
+    <string name="app_components_common_roundedCorners_label">Rounded corners</string>
```

Comment:
Maybe we should wait a final decision before adding this string to the translation file?

Reply:
I think we can add it because we might forget to add it if we wait for the final decision on this topic. By the way, the iOS key `app_components_button_rounded_label` has already been added to the file.